docs/server/source/server-reference/configuration.md

```diff
@@ -76,12 +77,13 @@ Note how the keys in the list are separated by colons.
 ```
 
 
-## database.host, database.port & database.name
+## database.backend, database.host, database.port & database.name
 
-The RethinkDB database hostname, port and name.
+The database backend to use (e.g. RethinkDB) and its hostname, port and name.
 
 **Example using environment variables**
 ```text
+export BIGCHAINDB_DATABASE_HOST=rethinkdb
```

Comment:
Fixed in https://github.com/bigchaindb/bigchaindb/pull/913/commits/6c9c0293a32075f0ebc91631d5d78b102d9e2097.